frontend-ssr/components/notification-tooltip/Notification.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Link from 'next/link';
+
+import { timeAgo } from '../../services/Utils';
+import styles from './Notification.module.scss';
+
+interface Props {
+  id: string,
```

Comment:
`id` is useless here. We're not using it inside this component so let's delete it.